# Request
RGBDepthImage combined_image
geometry_msgs/Pose2D position
---
# Response